### ISSUE TITLE
Add term intoeing

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10465]
**Preferred term label:**
Intoeing

**Synonyms**
pigeon-toed

**Definition (free text, please give PubMed ID)**
PMID: 29148202: Caused by a rotational variation anywhere in the lower extremity that causes the foot to point inward, instead of pointing straight ahead or outward, when walking or running. 
PMID: 28925669: The 3 major causes of pediatric in-toeing are metatarsus adductus, internal tibial torsion, and femoral anteversion. 

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0001288

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
For example, Desbuquois dysplasia (omim615777)  PMID22711505

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@lisannesw @azankl

Is this distinct from [Metatarsus adductus HP:0001840](https://hpo.jax.org/browse/term/HP:0001840)?

**Comment by azankl** [comment]
@pnrobinson yes, as it says in Lisanne's definition above, rotational variation anywhere in the lower extremity can lead to in-toeing, the 3 major causes of pediatric in-toeing are metatarsus adductus, internal tibial torsion, and femoral anteversion.

**Comment by pnrobinson** [comment]
@azankl 
Adding new term: 
- ID: HP:6001054
- label: Intoeing
- parents: [Gait disturbance (HP:0001288)](https://hpo.jax.org/app/browse/term/HP:0001288)
- pmid: PMID:28925669|PMID:29148202
- def: An abnormal gait caused by a rotational variation anywhere in the lower extremity that causes the foot to point inward, instead of pointing straight ahead or outward, when walking or running.
- comment: The three major causes of pediatric in-toeing are metatarsus adductus, internal tibial torsion, and femoral anteversion.
- orcid: orcid:0009-0007-1020-5412
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10465
- synonyms: Pigeon-toed(NONE)
